lib/manageiq/release.rb

```diff
@@ -9,6 +9,7 @@
 require 'manageiq/release/license'
 require 'manageiq/release/readme_badges'
 require 'manageiq/release/travis'
+require 'manageiq/release/internationalization'
```

Comment:
alphabetize

Reply:
Also this section are like the "libraries" as opposed to the "commands", so I think this should go below in the next requires section (I have a local change to organize that a bit better to make it clearer).